### PR TITLE
Refactored plots and experiment running

### LINKED ISSUE #62: Refactor plot generation
1. Rename LOAD_RATIO -> LOAD_VARIATION_RATIO
2. Rename experiments (we want to knwo what they represents)
3. Add to plots information about others parameters.
4. Eliminate duplicates in generate plot method

### PR DESCRIPTION
**Pull request by Bertt-hub**
- added function used in all experiments to run other functions (shard allocation, simulation, imbalance and delay calculation)
- added params near plots
- refractored plots

### CODE REVIEW

**Review by pientaa — CHANGES_REQUESTED** [review]
Solve conflicts

File: estimator/delay_estimator.py [hunk 1]
```diff
@@ -42,12 +42,12 @@ def estimate_delays(parallel_requests=5):
 
         T = pd.DataFrame(vectors).values.sum() * abs(node_load / (1 - node_load)) * (c_a * c_a / 2.0)
 
-        print("node_load:")
-        print(node_load)
-        print("c_a:")
-        print(c_a)
-        print("T:")
-        print(T)
+        # print("node_load:")
```
Comment (pientaa) [inline estimator/delay_estimator.py]: delete instead of comment

File: experiment_executor.py [hunk 2]
```diff
@@ -138,16 +139,15 @@ def experiment_one(self):
...
-                cloud_load_lvl = processing_time / (periods_in_vector * self.num_of_nodes * parallel_requests)
-
-                self.shard_allocation(algorithm). \
-                    calculate_imbalance_level(algorithm, CLOUD_LOAD_LEVEL, cloud_load_lvl). \
-                    simulation(algorithm). \
-                    calculate_delays(algorithm, CLOUD_LOAD_LEVEL, cloud_load_lvl). \
-                    estimate_delays(algorithm, CLOUD_LOAD_LEVEL, cloud_load_lvl)
+                self.run_experiment(algorithm, CLOUD_LOAD_LEVEL, cloud_load_lvl)
 
+        self.experiment_static_params = "Nodes: %d \nShards: %d \nLoad μ: %.2f \nLoad σ: %.2f " % (self.num_of_nodes,
```
Comment (pientaa) [inline experiment_executor.py]: this could a class method that adds this property

File: experiment_executor.py [hunk 3]
```diff
@@ -229,45 +234,27 @@ def save_delays_and_imbalance(self, experiment):
...
+
+        for index in range(3):
+            plt.clf()
+            for group in plot_params["dataframe"][index]['algorithm'].unique():
+                x = plot_params["dataframe"][index][plot_params["dataframe"][index]['algorithm'] == group][experiment].tolist()
+                y = plot_params["dataframe"][index][plot_params["dataframe"][index]['algorithm'] == group][plot_params["df_column"][index]].tolist()
+                plt.plot(x, y, label=group, linewidth=2)
+            path = "experiments/" + experiment + plot_params["path_folder"][index] + experiment + "_" + getCurrentDateTime()
+            plt.legend(loc="upper right")
+            # TODO: Change label to more readable representation
```
Comment (pientaa) [inline experiment_executor.py]: maybe this comment is not needed aymore?

Comment (Bertt-hub) [reply experiment_executor.py]: Deleted

File: experiment_executor.py (hunk 2 above)
Comment (Bertt-hub) [reply experiment_executor.py]: Added one class method

**Review by pientaa — APPROVED** [review]

**Pull request merged**